[OSDOCS#9601]: Release Notes 4.14.12 (#71558)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

[source,terminal]
----
$ oc adm release info 4.14.12 --pullspecs
----

[id="ocp-4-14-12-features"]
==== Features

The following feature is included in this z-stream release:

[id="ocp-4-14-12-configuring-linuxptp-gm"]
===== Using dual Intel E810 Westport Channel NICs as grandmaster clock with the PTP Operator

* You can now configure `linuxptp` services as grandmaster clock (T-GM) for dual Intel E810 Westport Channel NICs by creating a `PtpConfig` custom resource (CR) that configures both NICs.
The host system clock is synchronized from the NIC that is connected to the GNSS time source.
The second NIC is synced to the 1PPS timing output provided by the NIC that is connected to GNSS.
For more information see, xref:../networking/ptp/configuring-ptp.adoc#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp[Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs]. (link:https://issues.redhat.com/browse/TELCODOCS-1480[*TELCODOCS-1480*])

[id="ocp-4-14-12-bug-fixes"]
==== Bug Fixes

* Previously, the release-to-channel strategy and `oc-mirror` behavior caused an error with the selective mirroring of packages.

> 

Reviewer: ```suggestion
```

With this release, the issue has been resolved. 
You can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.
(link:https://issues.redhat.com/browse/OCPBUGS-28871[*OCPBUGS-28871*])

* Previously, the CPU limits from the EFS CSI driver container had the potential to cause performance degradation. With this release, the CPU limits from the EFS CSI driver container have been removed. (link:https://issues.redhat.com/browse/OCPBUGS-28823[*OCPBUGS-28823*])

* Previously, when using the `routingViaHost` mode, access to the `ExternalTrafficPolicy=Local` load balancer services broke. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28789[*OCPBUGS-28789*])

* Previously, when a HostedCluster was deployed and a user defined a KAS `AdvertiseAddress`, it conflicted with the current deployment, overlapping with the other networks like Service, Cluster or Machine network, which caused a deployment failure. With this release, network validations for `AdvertiseAddress` have been added. (link:https://issues.redhat.com/browse/OCPBUGS-20547[*OCPBUGS-20547*])

[id="ocp-4-14-12-updating"]
==== Updating

> To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

Reviewer: ```suggestion
To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Author: good catch! tysm!

$ oc adm release info 4.14.0 --pullspecs
----
//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.

[id="ocp-4-14-12"]
=== RHSA-2024:0735 - {product-title} 4.14.12 bug fix and security update

Issued: 2024-02-13

{product-title} release 4.14.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0735[RHSA-2024:0735] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0738[RHBA-2024:0738] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.12 --pullspecs
----

[id="ocp-4-14-12-features"]
==== Features

The following feature is included in this z-stream release:

[id="ocp-4-14-12-configuring-linuxptp-gm"]

> ===== Enabling configuration of Linux PTP services 

Reviewer: ```suggestion
===== ===== Using dual Intel E810 Westport Channel NICs as grandmaster clock with the PTP Operator
```

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.12 --pullspecs
----

[id="ocp-4-14-12-features"]
==== Features

The following feature is included in this z-stream release:

[id="ocp-4-14-12-configuring-linuxptp-gm"]
===== Enabling configuration of Linux PTP services 

* You can now configure `linuxptp` services as grandmaster clock (T-GM) for dual Intel E810 Westport Channel NICs by creating a `PtpConfig` custom resource (CR) that configures both NICs.
The host system clock is synchronized from the NIC that is connected to the GNSS time source.
The second NIC is synced to the 1PPS timing output provided by the NIC that is connected to GNSS.
For more information see, xref:../networking/ptp/configuring-ptp#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp[Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs]. (link:https://issues.redhat.com/browse/TELCODOCS-1480[*TELCODOCS-1480*])

[id="ocp-4-14-12-bug-fixes"]
==== Bug Fixes

> * Previously, the release-to-channel strategy and `oc-mirror` behavior was causing issues for users that perform selective mirroring of our packages. When a user performed selective mirroring of the most recent (and hence default) channel for their package and a new release introduced a new channel, their current default channel became invalid and automatically assigning a new default channel failed. With this release, users can define a `defaultChannel` in the `ImageSetConfig` yaml file, which overrides the `currentlDefault` channel. (link:https://issues.redhat.com/browse/OCPBUGS-28871[*OCPBUGS-28871*])

Reviewer: Edit for clarity

```suggestion
* Previously, the release-to-channel strategy and `oc-mirror` behavior caused an error with the selective mirroring of packages.
When you selectively mirror the most recent (and hence default) channel for the package and a new release introduces a new channel, the current default channel becomes invalid and the automatic assignment of the new default channel fails.
This problem is now resolved. 
You can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.
(link:https://issues.redhat.com/browse/OCPBUGS-28871[*OCPBUGS-28871*])
```

$ oc adm release info 4.14.0 --pullspecs
----
//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.

> 

Reviewer 2: Remove whitespace?
